modules/@apostrophecms/schema/index.js

```diff
@@ -142,11 +142,14 @@ module.exports = {
       // leading slash required). Otherwise, expect a object-style slug
       // (no slashes at all)
       convert: function (req, field, data, destination) {
-        const options = {};
+        const options = {
+          def: field.def !== undefined ? field.def : undefined
```

Comment:
Seems like this must be equivalent to `def: field.def` since you handle undefined by passing undefined.

Reply:
oh, true.